objective.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, l1_ratio=.5, reg=1.0, fit_intercept=False):
     def set_data(self, X, y):
         self.X, self.y = X, y
         self.n_samples, self.n_features = self.X.shape
-        self.lmbda_max = np.max(np.abs(X.T @ y)) / len(y)
+        self.lmbda_max = np.max(np.abs(X.T @ y)) / (len(y) * self.l1_ratio)
```

Comment:
Shouldn't `lambda_max` be adjusted if the intercept is fit as in https://github.com/benchopt/benchmark_lasso/blob/ff69e6b0bf9ef8e172346000fabf87489de84b9f/objective.py#L40-L44 ?

Reply:
yes, you are right.

Reply:
thanks @jolars